cdap-ui/app/tracker/templates/dictionary.html

```diff
@@ -0,0 +1,89 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+  
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+  
+  http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<div class="row">
+  <div class="col-xs-12">
+    <label class="heading">Data Dictionary</label>
+  </div>
+</div>
+<div class="row">
+  <div class="col-sm-12">
+    <div ng-if="DictionaryController.dictionaryData.length > 0"
```

Comment:
It might be good to initialize with an empty row, so the user know they can input something